src/components/recordingcreator/recordingeditor.js

```diff
@@ -1,150 +1,150 @@
-    'use strict';
-
-    scrollHelper = scrollHelper.default || scrollHelper;
-    loading = loading.default || loading;
-
-    var currentDialog;
-    var recordingDeleted = false;
-    var currentItemId;
-    var currentServerId;
-    var currentResolve;
-
-    function deleteTimer(apiClient, timerId) {
-        return new Promise(function (resolve, reject) {
-            require(['recordingHelper'], function (recordingHelper) {
-                recordingHelper = recordingHelper.default || recordingHelper;
-
-                recordingHelper.cancelTimerWithConfirmation(timerId, apiClient.serverId()).then(resolve, reject);
-            });
+import dialogHelper from 'dialogHelper';
+import globalize from 'globalize';
+import layoutManager from 'layoutManager';
+import connectionManager from 'connectionManager';
+import require from 'require';
+import loading from 'loading';
+import scrollHelper from 'scrollHelper';
+import 'scrollStyles';
+import 'emby-button';
+import 'emby-collapse';
+import 'emby-input';
+import 'paper-icon-button-light';
+import 'css!./../formdialog';
+import 'css!./recordingcreator';
+import 'material-icons';
+import 'flexStyles';
+
+let currentDialog;
+let recordingDeleted = false;
+let currentItemId;
+let currentServerId;
+let currentResolve;
+
+function renderTimer(context, item, apiClient) {
+    context.querySelector('#txtPrePaddingMinutes').value = item.PrePaddingSeconds / 60;
+    context.querySelector('#txtPostPaddingMinutes').value = item.PostPaddingSeconds / 60;
+
+    loading.hide();
+}
+
+function closeDialog(isDeleted) {
+    recordingDeleted = isDeleted;
+    dialogHelper.close(currentDialog);
+}
+
+function onSubmit(e) {
+    const form = this;
+
+    const apiClient = connectionManager.getApiClient(currentServerId);
+
+    apiClient.getLiveTvTimer(currentItemId).then(function (item) {
+        item.PrePaddingSeconds = form.querySelector('#txtPrePaddingMinutes').value * 60;
+        item.PostPaddingSeconds = form.querySelector('#txtPostPaddingMinutes').value * 60;
+        apiClient.updateLiveTvTimer(item).then(currentResolve);
+    });
+
+    e.preventDefault();
+
+    // Disable default form submission
+    return false;
+}
+
+function init(context) {
+    context.querySelector('.btnCancel').addEventListener('click', function () {
+        closeDialog(false);
+    });
+
+    context.querySelector('.btnCancelRecording').addEventListener('click', function () {
+        const apiClient = connectionManager.getApiClient(currentServerId);
+
+        (apiClient, currentItemId).then(function () {
```

Comment:
What's going on here?

Reply:
Looks like I deleted a function, probably when rebasing :/

Reply:
Should be fixed now.